### ISSUE TITLE
More recent docker

### ISSUE THREAD

**Issue by Gandulf78** [WDCommunity/wdpksrc#111]
Hi. 
Could you please release a more recent Docker version? I tried to compile docker 20.10.20 with the scripts on my wdex2ultra but did not succeed (compilation error : non arm/v7 binaries). 
Thank you.

**Comment by darodi** [comment]
@Gandulf78 

Have you tried this?
https://github.com/JediNite/docker-ce-WDEX4100-binaries

**Comment by Gandulf78** [comment]
> Have you tried this?
> https://github.com/JediNite/docker-ce-WDEX4100-binaries

Yes. Finally I succeeded by building docker 20.17.0 on my Mac (not on the WDMyCloudEX2Ultra) following this advice 

http://postbits.de/syncthing.html

**Comment by JediNite** [comment]
> I have also noticed that the patch need be to adapted for the latest docker versions).

@Gandulf78 , yeah seems versions newer then that build don't work.  I had to use an older version of the debian OS within the docker containers for the build previously up to that version, but it looks like in the docker source code now ,it does not work with the older versions of debian for building anymore.  Not found a workaround for this yet unfortunately.
